cg/services/order_validation_service/rules/sample/rules.py

```diff
@@ -221,12 +223,33 @@ def validate_sample_names_available(
     return errors
 
 
+def validate_non_control_sample_names_available(
+    order: OrderWithControlSamples, store: Store, **kwargs
+) -> list[SampleNameNotAvailableError]:
+    """
+    Validate that non-control sample names do not exists in the database under the same customer.
```

Comment:
potential code duplication, I'm working in a way of using an utils functionto call here and in the function above `validate_sample_names_available`

Reply:
solved